<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateBook</class>
 <widget class="QMainWindow" name="UpdateBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>310</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>990</width>
    <height>310</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>990</width>
    <height>310</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Actualización de algún libro</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/book-flat.ico</normaloff>:/book-flat.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gbMain">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>671</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Raleway</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 244, 242); color: rgb(35, 54, 86);</string>
    </property>
    <property name="title">
     <string>Información</string>
    </property>
    <widget class="QLabel" name="lbTitle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Título</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbPageNumber">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>30</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Número de páginas</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lneTitle">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>28</y>
       <width>281</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="maxLength">
      <number>110</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbPageNumber">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>28</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="lbGenero">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Genero</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lneGenero">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>68</y>
       <width>311</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cboGenero">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>67</y>
       <width>261</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cboPublisher">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>99</y>
       <width>261</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnePublisher">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>100</y>
       <width>311</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbPublisher">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>102</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Editorial</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cboFormate">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>129</y>
       <width>261</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbFormate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>132</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Formato</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lneFormate">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>130</y>
       <width>311</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbCopiesNumber">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>167</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="lbCopiesNumber">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>147</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Número de ejemplares</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbEdition">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>30</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>15</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="lbPageNumber_2">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>32</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Edición</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbActions">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>291</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Raleway</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 244, 242); color: rgb(35, 54, 86);</string>
    </property>
    <property name="title">
     <string>Acciones</string>
    </property>
    <widget class="QPushButton" name="pbUpdate">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#pbUpdate {
background-color: rgb(123, 155, 166);
color:  rgb(255, 255, 255);
border-radius: 5px;
 }

 QPushButton#pbUpdate:hover {
background-color: rgb(65, 91, 118);
color:  rgb(255, 255, 255);
 }</string>
     </property>
     <property name="text">
      <string>   Actualizar</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/update_contact_info.ico</normaloff>:/update_contact_info.ico</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pbExit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#pbExit {
background-color: rgb(123, 155, 166);
color:  rgb(255, 255, 255);
border-radius: 5px;
 }

 QPushButton#pbExit:hover {
background-color: rgb(65, 91, 118);
color:  rgb(255, 255, 255);
 }</string>
     </property>
     <property name="text">
      <string>   Salir</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/door_out.ico</normaloff>:/door_out.ico</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbISBN">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>20</y>
      <width>671</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 244, 242); color: rgb(35, 54, 86);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="lbISBN">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>22</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ISBN</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lneISBN">
     <property name="geometry">
      <rect>
       <x>43</x>
       <y>20</y>
       <width>441</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="maxLength">
      <number>30</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pbSearch">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>10</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#pbSearch {
background-color: rgb(123, 155, 166);
color:  rgb(255, 255, 255);
border-radius: 5px;
 }

 QPushButton#pbSearch:hover {
background-color: rgb(65, 91, 118);
color:  rgb(255, 255, 255);
 }</string>
     </property>
     <property name="text">
      <string>Buscar información</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-16</x>
      <y>-15</y>
      <width>1021</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(65, 91, 118);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>gbMain</zorder>
   <zorder>gbActions</zorder>
   <zorder>gbISBN</zorder>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbExit</sender>
   <signal>clicked()</signal>
   <receiver>UpdateBook</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>816</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>704</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
